src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -0,0 +1,228 @@
+  withSystemContext(NewSearchBarInput),
+  {
+    viewer: graphql`
+      fragment NewSearchBarInput_viewer on Viewer
+        @argumentDefinitions(
+          term: { type: "String!", defaultValue: "" }
+          hasTerm: { type: "Boolean!", defaultValue: false }
+        ) {
+        searchConnection(query: $term, mode: AUTOSUGGEST, first: 7)
+          @include(if: $hasTerm) {
+          edges {
+            node {
+              displayLabel
+              href
+              imageUrl
+              __typename
+              ... on SearchableItem {
+                displayType
+                slug
+              }
+              ... on Artist {
+                statuses {
+                  artworks
+                  auctionLots
+                }
+              }
+            }
+          }
+        }
+        ...NewSearchInputPills_viewer @arguments(term: $term)
+      }
+    `,
+  },
+  graphql`
+    query NewSearchBarInputRefetchQuery($term: String!, $hasTerm: Boolean!) {
+      viewer {
+        ...NewSearchBarInput_viewer @arguments(term: $term, hasTerm: $hasTerm)
+      }
+    }
+  `
+)
+
+interface NewSearchBarInputQueryRendererProps extends BoxProps {
+  isXs: boolean
+}
+
+export const NewSearchBarInputQueryRenderer: FC<NewSearchBarInputQueryRendererProps> = props => {
+  const { relayEnvironment, searchQuery = "" } = useContext(SystemContext)
+  const isMounted = useDidMount(typeof window !== "undefined")
+
+  if (!isMounted) {
+    return <StaticSearchContainer searchQuery={searchQuery} {...props} />
+  }
+
+  return (
+    <SystemQueryRenderer<NewSearchBarInputSuggestQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query NewSearchBarInputSuggestQuery(
+          $term: String!
+          $hasTerm: Boolean!
+        ) {
+          viewer {
+            ...NewSearchBarInput_viewer
+              @arguments(term: $term, hasTerm: $hasTerm)
+          }
+        }
+      `}
+      variables={{
+        hasTerm: false,
+        term: "",
+      }}
+      render={({ props: relayProps }) => {
+        if (relayProps) {
+          return (
+            <NewSearchBarInputRefetchContainer
+              viewer={relayProps.viewer}
+              isXs={props.isXs}
+            />
+          )
+          // SSR render pass. Since we don't have access to `<Boot>` context
+          // from within the NavBar (it's not a part of any app) we need to lean
+          // on styled-system for showing / hiding depending upon breakpoint.
+        } else {
+          return <StaticSearchContainer searchQuery={searchQuery} />
+        }
+      }}
+    />
+  )
+}
+
+/**
+ * Displays during SSR render, but once mounted is swapped out with
+ * QueryRenderer below.
+ */
+const StaticSearchContainer: FC<{ searchQuery: string } & BoxProps> = ({
+  searchQuery,
+  ...rest
+}) => {
+  const { t } = useTranslation()
+  return (
+    <>
+      <Box display={["block", "none"]} {...rest}>
+        <SearchInputContainer
```

Comment:
**question**: do we need to use prev `SearchInputContainer` component here?

Reply:
you mean that I should create a new `NewSearchInputContainer` or that we don't need it at all?